packages/common/src/intl/locales/en/distribution.json

```diff
@@ -98,7 +97,7 @@
   "messages.deleted-requisitions": "Deleted {{number}} requisitions",
   "messages.error-saving-shipment": "Error saving shipment 🥺",
   "messages.must-allocate-all-lines": "Cannot change the status until all lines have been allocated.",
-  "messages.select-rows-to-delete": "Select rows to delete",
+  "messages.select-rows-to-delete-them": "Select rows to delete them",
```

Comment:
offroad - tidy up these. there was some duplication here; have made the delete message consistent across the translation files.